docs/wiki/Cryptography/bls.md

```diff
@@ -0,0 +1,232 @@
+<!-- @format -->
+
+# Boneh-Lynn-Shacham (BLS) signature
+
+### TLDR;
+
+- Proof of stake protocols use digital signatures to identify their participants and hold them accountable.
+- BLS signatures can be aggregated together, making them efficient to verify at large scale.
+- Signature aggregation allows the beacon chain to scale to hundreds of thousands of validators.
+- Ethereum transaction signatures on the execution (Eth1) layer remain as-is.
+
+BLS is a digital signature scheme with aggregation properties. Given set of signatures (_signature_1_, ..., _signature_n_) anyone can produce an aggregated signature. Aggregation can also be done on secret keys and public keys. Furthermore, the BLS signature scheme is deterministic, non-malleable, and efficient. Its simplicity and cryptographic properties allows it to be useful in a variety of use-cases, specifically when minimal storage space or bandwidth are required. This page will cover general idea and Math behind BLS signatures, further cover BLS in context of Ethereum.
+
+## How BLS works?
+
+At the core of BLS signatures is the concept of bilinear mapping through pairings on elliptic curves. A key component is a pairing function $e$, defined between two groups derived from elliptic curves:
+
+$$e: G_1 \times G_2 \rightarrow G_T$$
+
+This function is efficiently computable and must satisfy bilinear properties:
+
+- For all $P,Q$ in $G_1$ and $a$ in integers, bilinearity is defined as:
+  $$ e(aP, Q) = e(P, Q)^a $$
+  $$ e(P, aQ) = e(P, Q)^a $$
+
+- Additionally, it must distribute over addition:
+  $$ e(P + Q, R) = e(P, R) \times e(Q, R) $$
+  $$ e(P, Q + R) = e(P, Q) \times e(P, R) $$
+
+These properties enable the cryptographic mechanisms necessary for functions like signature aggregation, which is a pivotal feature in blockchain applications and cryptographic consensus.
+
+#### Transition from ECDSA to BLS
+
+Traditional ECDSA signatures, as commonly used in Bitcoin, depend heavily on the randomness of number generation and necessitate verification of all involved public keys, which can be computationally intensive. This prompted a shift towards the Schnorr signature scheme, which allows for some aggregation but still lacks the full efficiencies gained from BLS.
```

Comment:
May be explaining a bit on its implications for network security and validator efficiency can really show the motivation behind the transitioning to BLS signatures. For example: 

In Ethereum PoS, validators produce "attestations" which are votes for a block that they believe should be added to the beacon chain. These attestations are frequent and numerous; hence, using BLS allows these attestations to be aggregated into a single signature. For instance, instead of having to process and verify 10,000 individual signatures from 10,000 validators, the network can process a single aggregated signature. This dramatically reduces computational overhead and storage requirements on the chain, leading to faster processing times and reduced costs.

Reply:
you're right, I've mentioned the advantages of using BLS but was thinking to deep dive on the use BLS in Ethereum in CL page. What do you think?